packages/buefy-next/src/components/progress/Progress.spec.ts

```diff
@@ -1,22 +1,26 @@
 import { shallowMount } from '@vue/test-utils'
-import BProgress from '@components/progress/Progress'
+import type { VueWrapper } from '@vue/test-utils'
+import BProgress from '@components/progress/Progress.vue'
+import { describe, expect, it } from 'vitest'
+
+let wrapper: VueWrapper<InstanceType<typeof BProgress>>
```

Comment:
I do not think there would be any benefit of hoisting scoped `const wrapper`s to a shared `let wrapper`. Scoping a variable or constant is generally better.

Reply:
True. I thought it would be best simply because a majority of our unit tests do use a shared let wrapper

Reply:
Although most of the unit tests did use a shared `let wrapper` that I did not like, I left them to keep the original code unchanged as much as possible. Because changing the scope of a variable might alter the behavior. Sorry for confusing you.

Reply:
> Although most of the unit tests did use a shared `let wrapper` that I did not like, I left them to keep the original code unchanged as much as possible. Because changing the scope of a variable might alter the behavior. Sorry for confusing you.

Now that I think about it though I can't really see how this would affect the test unless there is some off chance the order of how the test is performed is not top-down